### ISSUE TITLE
Error handling incomplete / or its bugged

### ISSUE THREAD

**Issue by erikkubica** [api-platform/api-platform#1508]
Hi there,

I wanted to translate the Not Found (and also other default messages) I came across this page https://api-platform.com/docs/core/errors/  and followed the instructions, sadly after about 2 hours of mindf*k I figured out by accident that problem was, that this code executes only if entity is present in the database. But when you query (using request) a entity that does not exists at all it still returns the default Not Found error message.

So I would like to see how this situation is properly handled.

**Comment by dunglas** [comment]
I don't think that these technical errors are translatable. It's not supported by Symfony, nor API Platform. Actually, they are technical error, I don't get the point of translating them. They should probably be mapped to user-friendly error messages before being displayed to end users.

**Comment by erikkubica** [comment]
Thanks for the reply. I will then do the mapping thing, I did not want to but I will have to. It would be a nice feature tho to be able to override / hook those things since if someone doing multilingual API it's nice touch to return errors in the given language. Thanks anyway

**Comment by soyuka** [comment]
Closing as stale per #2987 (no activity since 2020-04-21). This repository is transitioning to a CLI installer. If this still reproduces against a current release of API Platform, please open a new issue on `api-platform/core` with a minimal reproduction.
